.github/workflows/release-images.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
     name: Release on GitHub
     runs-on: ubuntu-20.04
     container:
-      image: 1.16.2.1_focal
+      image: golang:1.16.2.1_focal
```

Comment:
Could you tell me why you didn't use `actions/setup-go`?

Reply:
This release workflow is using `ghr` to GitHub release because [the official action](https://github.com/actions/create-release) to release was archived.

`quay.io/cybozu/golang:1.16-focal` container already contains `ghr`, so I thought releasing would be easy if we use the image as the job's base image. 
(`golang:1.16.2.1_focal` is a mistake)